### PR TITLE
eris 0.11.4 (new formula)

### PR DESCRIPTION
**Pull request by pietv**
- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew-core/blob/master/.github/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally prior to submission with `brew install <formula>` (where `<formula>` is the name of the formula you're submitting)?
- [x] Does your submission pass `brew audit --strict --online <formula>` (after doing `brew install <formula>`)?

---

[Eris](https://erisindustries.com) blockchains application platform command line tools.

Go resources are "vendored" in, so no `go_resource` statements in the formula.

### CODE REVIEW

File: Formula/eris.rb [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  url "https://github.com/eris-ltd/eris-cli/archive/v0.11.4.tar.gz"
+  sha256 "e2eb02d01b76e8be9f28aac31b2e56ebadc4d0decb21fcfefb2f219cb03d1238"
+
+  depends_on "go" => :build
+  depends_on "docker"
+  depends_on "docker-machine"
+
+  def install
+    ENV["GOPATH"] = buildpath
+    ENV["GOBIN"] = bin
```
Comment (bfontaine) [inline Formula/eris.rb]: Is setting `GOBIN` really necessary to build that?

File: Formula/eris.rb [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+    (buildpath/"src/github.com/eris-ltd").mkpath
+    ln_sf buildpath, buildpath/"src/github.com/eris-ltd/eris-cli"
+
+    system "go", "build", "-o", "#{bin}/eris", "github.com/eris-ltd/eris-cli/cmd/eris"
+  end
+
+  test do
+    system "#{bin}/eris", "version"
```
Comment (bfontaine) [inline Formula/eris.rb]: Can’t we write a test that does something more substantial than printing the version?

Comment (pietv) [inline Formula/eris.rb]: Eris is a Docker container orchestrator and to do something substantial I will need the user to start Docker and Docker Machine first. I don't want to require them to do that for just an install.

Many formulas I saw seem to just check the version.

File: Formula/eris.rb (hunk 1 above)
Comment (pietv) [inline Formula/eris.rb]: You're right: it's not necessary. Will remove.

File: Formula/eris.rb [hunk 3]
```diff
@@ -0,0 +1,25 @@
+class Eris < Formula
+  desc "Blockchain application platform CLI"
+  homepage "https://erisindustries.com"
+  url "https://github.com/eris-ltd/eris-cli/archive/v0.11.4.tar.gz"
+  sha256 "e2eb02d01b76e8be9f28aac31b2e56ebadc4d0decb21fcfefb2f219cb03d1238"
+
+  bottle :unneeded
```
Comment (DomT4) [inline Formula/eris.rb]: You'll need bottles here...

Comment (pietv) [inline Formula/eris.rb]: You mean i will need something like this:
```
  bottle do
    cellar :any_skip_relocation
    sha256 "42a60235f9f31d12b59d33867b4f74ac4e4fa979fd92f7066bd5720da2eb607b" => :el_capitan
    sha256 "7acb5fcbd1230614a630cb4c0fefaef4a40b66dba633e067a641c17a4d444db2" => :yosemite
    sha256 "7cc1cf98d45f0cfd43e0d822d60430670e58dbd40f221033b2a446f0f03bb681" => :mavericks
  end
```

Do you recommend to drop the statement?

Comment (DomT4) [inline Formula/eris.rb]: Just remove the `bottle :unneeded` line. Our CI will automagically add bottles if/when merged.

Comment (pietv) [inline Formula/eris.rb]: Done.

File: Formula/eris.rb (hunk 2 above)
Comment (mikemcquaid) [inline Formula/eris.rb]: This is failing: http://bot.brew.sh/job/Homebrew%20Core%20Pull%20Requests/1894/version=el_capitan/testReport/junit/brew-test-bot/el_capitan/test_eris/

Comment (mikemcquaid) [inline Formula/eris.rb]: Probably need to use `shell_output` with `1` as the second parameter to check for that exit code, `2>&1` to redirect stderr to stdout and `assert_match` to check some output on the message.

Comment (pietv) [inline Formula/eris.rb]: Thanks. Yeah, I know what that error means, I've changed the flag.

**Comment by pietv** [comment]
Is anything pending from me before the merge?

**Comment by mikemcquaid** [comment]
Thanks for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

**Pull request closed without merging**